README.md

```diff
@@ -16,7 +16,9 @@ the needs of your specific game and team.
 ## History
 
 This framework was initially developed for the adventure game
-[The Interactive Adventures of Dog Mendonça and Pizzaboy®](http://store.steampowered.com/app/330420)
+*The Interactive Adventures of Dog Mendonça & Pizzaboy®*
+([full game](https://store.steampowered.com/app/330420),
+[demo](https://archive.org/details/Dog_Pizzaboy_demo))
```

Comment:
This is actually a demo with source code included, this might be worth mentioning (see files list at https://ia600202.us.archive.org/view_archive.php?archive=/18/items/Dog_Pizzaboy_demo/Dog_Pizzaboy_demo.zip). Not sure what Godot version it's compatible with though, if it's pre-1.0 it might be a bit tricky to use).

Reply:
I'm purposely giving a mere pointer to what is more or less abandonware, with all the associated legal grey area.